draft-ietf-quic-multipath.md

```diff
@@ -240,12 +240,11 @@ ACK frames when used with the multipath extension acknowledge packets for the pa
 As multipath support is unknown during the handshake, acknowledgments of
 Initial and Handshake packets are sent using ACK frames.
 
-After the handshake concluded and both endpoints support the multipath extension,
+After the handshake concluded with support for the multipath extension,
 endpoints SHOULD use PATH_ACK frames instead of ACK frames,
-including for acknowledging so far unacknowledged 0-RTT packets using Path ID 0.
-Endpoints MUST process ACK frames that acknowledge 0-RTT packets or 1-RTT packets,
-even when this extension is used. For example,
-ACK frames might be preferred by the sender as long as only the initial path
+also so far unacknowledged 0-RTT packets using Path ID 0.
+Endpoints MUST still process ACK frames that acknowledge 0-RTT packets or 1-RTT packets.
+For example, ACK frames might be preferred by the sender as long as only the initial path
 with Path ID 0 is in use.
```

Comment:
I also agree that this wording is a bit strange. I am not sure to understand the "only" here.
```suggestion
For example, ACK frames might be preferred by the sender to acknowledge packets
of the initial path with Path ID 0.
```

Reply:
Actually this changes the meaning of the example. I though the case that was brought up is that you might want to negotiate multipath support (for potential later use) but be able to have a quic connection that is behaving completely in the same way as a single path connection as long as you allow only for one path. That's why it says "only".

Reply:
With the provided explanation, this somewhat makes sense. However, without such context, I find this sentence a bit confusing. But this is not blocking.

Reply:
@qdeconinck any proposal how to improve the sentence?

Reply:
My proposal would be to add more context to the example, but the "example text" would start being larger. I will do a suggestion.

```suggestion
For example, a sender may negotiate multipath support for later use and keep
only the initial path with path ID 0 for a while. During this single-path period,
ACK frames might be preferred by the sender.
```

Reply:
wpm